js/RichText/FormTags.js

```diff
@@ -85,7 +85,7 @@ GLPI.RichText.FormTags = class
     }
 
     async #fetchItems(filter) {
-        const url = CFG_GLPI.root_doc + '/Form/TagList';
+        const url = CFG_GLPI.root_doc + '/form/tag-list';
```

Comment:
When the URL will correspond to classes names, it will be preferable to use the correct case (to not have to deal with case sensitivity issues when loading the class), and the result will be URLs like `/Ticket/{$id}`.
Therefore, I think that we should use camel case in our new URLs everywhere, to not have a mix between kebab case and camel case.